nservicebus/upgrades/5to6.md

```diff
@@ -664,3 +664,9 @@ NOTE: This ID needs to be stable and should never be hardcoded, e.g. it can be r
 Explicitly setting property values via `.ConfigureProperty<T>()` and `.InitializeHandlerProperty<T>()` has been deprecated. Instead configure the properties explicitly using:
 
 snippet: 5to6-ExplicitProperties
+
+## IWantToRunWhenBusStartsAndStops is obsolete
+
+The `IWantToRunWhenBusStartsAndStops` interface is no longer supported. When self-hosting, call any startup code after `Endpoint.Create` or any cleanup code after `Endpoint.Stop`. When using either NServiceBus.Host or NServiceBus.Host.AzureCloudService, use the host's `IWantToRunWhenEndpointStartsAndStops` interface.
+
+For tests that implement `IWantToRunWhenBusStartsAndStops`, use a `FeatureStartupTask`.
```

Comment:
Is there documentation to FeatureStartupTask or an associated sample to link to?